src/components/03-components/03-buttons/02-circle/circle.njk

```diff
@@ -0,0 +1,33 @@
+{{name}}{%- if (name == 'Outline-inverse') -%} 
+{% set bkg = "bg-base-darkest text-base-lightest " -%} 
+{% endif -%} 
+{%- if (name == 'White') -%} 
+{% set bkg = "bg-base-lightest " -%} 
+{% endif -%} 
+<div class="{{ bkg }}padding-4">
+  <div class="grid-row">
+    <div class="grid-col">
+      <div class="padding-y-2"><div class="text-center">{{name}}<br>{% render '@button--single', button %}  </div></div>
```

Comment:
similar to the text color class, I would remove text-center from wrapping icon (so just for the title)